applications/nrf5340_audio/src/audio/streamctrl_broadcast_sink.c

```diff
@@ -352,11 +267,11 @@ static void button_msg_sub_thread(void)
 				}
 
 				break;
-			} else if (IS_ENABLED(CONFIG_BT_BAP_BROADCAST_SINK)) {
-				/* Will eventually be handled by different applications */
-				ret = le_audio_disable();
+			} else {
+				ret = broadcast_sink_disable();
 				if (ret) {
-					LOG_ERR("Failed to disable the broadcast receiver");
+					LOG_ERR("Failed to disable the broadcast sink");
```

Comment:
Would be great if we print ret in here.

Reply:
Done

Reply:
Probably we missed one in here :P

Reply:
Ah, sorry! I thought this comment was a duplicate. I will fix

Reply:
Done